### PR TITLE
fixed issue #884

### PR DESCRIPTION
**Pull request by monishamandal02**
I had done all the modifications which had been asked. Please check :)

### CODE REVIEW

**Comment by smaranjitghose** [comment]
# Feedback:

The background color of the cards are to be changed to a shade of black/gray for dark mode as well

### Current Version

Change_color_of_cards

### Sample Requirement:

Colot_Tr

[Hacktoberfest Reference](https://hacktoberfest.digitalocean.com/)

File: assets/css/education.css [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
     --shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
     --border: 1px solid #ecf0f3;
+    --font-color: black;
 }
 
 html[data-theme='dark'] {
-    --bg1-color: #272727;
-    --shadow: 0 4px 8px 0 rgba(104, 102, 102, 0.2);
-    --border: none;
+    --bg1-color: #000033;
```
Comment (smaranjitghose) [inline assets/css/education.css:17]: Please have a look at the reference I gave, inspect the website and get the colors. The entire point of changing this is to make sure that the cards are visible separately from the background color

File: assets/js/education.js [hunk 2]
```diff
@@ -115,7 +115,7 @@ const showCards = () => {
     (output += `        
         <div class="col-6 col-md-3 col-sm-4 column" data-aos="fade-up" data-aos-easing="linear" data-aos-delay="600" >  
             <div class="card mb-3 mx-auto">
-               <div class="content">
+            <div class="content">
```
Comment (smaranjitghose) [inline assets/js/education.js:118]: Unnecessary Indentation Change. Please revert this

File: assets/css/education.css [hunk 3]
```diff
@@ -1,28 +1,35 @@
 @import url(./style.css);
+@font-face {
+    font-family: "NHaas";
```
Comment (smaranjitghose) [inline assets/css/education.css:3]: Remove this and use Inter Font please

File: assets/css/education.css [hunk 4]
```diff
@@ -1,28 +1,35 @@
...
 
 html[data-theme='dark'] {
-    --bg1-color: #272727;
-    --shadow: 0 4px 8px 0 rgba(104, 102, 102, 0.2);
-    --border: none;
+    --bg1-color: #000033;
+    --shadow: 0 4px 8px 0 rgb(255, 255, 255);
+    --border: 1px solid rgb(0, 57, 230);
+    --font-color: #fff;
 }
```
Comment (smaranjitghose) [inline assets/css/education.css:21]: Declare one variable for background color and another for card color

File: assets/css/education.css [hunk 5]
```diff
@@ -74,42 +83,51 @@ html[data-theme='dark'] {
...
     font-weight: 450;
     font-size: 2.5vh;
+    color: var(--font-color);
 }
 
 .s2 .card img {
-    height: 90%;
+    height: 80%;
     width: 90%;
+    color: var(--font-color);
```
Comment (smaranjitghose) [inline assets/css/education.css:111]: the Font color is not be used for this 😅

File: assets/css/education.css [hunk 6]
```diff
@@ -187,8 +205,8 @@ p {
 
 #mooc .card {
     border-radius: 5px;
-    background: #ecf0f3;
-    box-shadow: 0 6px 10px rgba(0, 0, 0, 0.08), 0 0 6px rgba(0, 0, 0, 0.05);
+    background-color: var(--bg1-color);
```
Comment (smaranjitghose) [inline assets/css/education.css:208]: Use card color for this and not the background color

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
Please define the variables in the CSS files for the following and use the corresponding values as specified

- Background Color: #072540 (Dark Mode)
- Element Color: #93c2db (Dark Mode)
- Font Color: #0069ff (Light Mode)
- Illustration Color: #0069ff (Light Mode)


- Font: Inter

File: assets/css/education.css [hunk 7]
```diff
@@ -1,28 +1,35 @@
 @import url(./style.css);
+@font-face {
+    font-family: "NHaas";
+    src: url(../fonts/NHaas\ Bold.63b9aba.woff2);
+}
 
 /* Dark and Light theme */
```
Comment (smaranjitghose) [inline assets/css/education.css:7]: This is a self-explanatory part of the stylesheet. Could you please remove this comment and all spaces before this

**Comment by smaranjitghose** [comment]
color_shadow
Do not add a color to the shadow please

**Comment by smaranjitghose** [comment]
@monishamandal02 I have already given the feedback for this PR. Please incorporate the suggested changes and let me know.

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Box shadow is too large avoid that cause it looks like a black border rather than shadow

**Comment by smaranjitghose** [comment]
@monishamandal02 Any updates?

**Comment by monishamandal02** [comment]
I have removed the extra changes like the blue shadow, fonts and all. I am lagging with some knowledge I couldn't switch the colour of cards, mooc cards after using the dark-light switch.

**Comment by anushbhatia** [comment]
> I have removed the extra changes like the blue shadow, fonts and all. I am lagging with some knowledge I couldn't switch the colour of cards, mooc cards after using the dark-light switch.

Could you share exactly where are you lagging and contact us on the relevant platform to resolve this. Also for dark mode different colour variables are already defined just check how to use them

**Pull request closed without merging**